doc/nrf/device_guides/working_with_nrf/nrf70/developing/scan_operation.rst

```diff
@@ -0,0 +1,313 @@
+
+* Band filtering: Controls the Wi-Fi frequency bands to be scanned (for example, 2.4 GHz, 5 GHz, or both) on the nRF7000 and nRF7002 ICs.
+
+* Forced passive scan: Forces a passive scan on all channels.
+
+* Active dwell time control: Controls the time spent scanning on an active channel.
+
+* Passive dwell time control: Controls the time spent scanning on a passive channel.
+
+* Channel control: Controls the channels to be scanned.
+
+* SSID filtering: Controls the SSIDs to be scanned.
+
+* Scan result limit control: Controls the number of scan results returned to an application.
+
+The above controls can be used individually or in combination to achieve:
+
+* Optimized scanning behavior - It can be used by applications based on their requirements and the conditions of the environment where an end product is expected to be deployed.
+
+* Reduced processing overhead
+
+* Faster scan times
+
+* Reduced power consumption
+
+Scan API
+********
+
+The scan API can be used to implement features such as scan functionality and controlled scan provided by the nRF70 Series ICs.
+
+You can find the scan API reference in :ref:`zephyr:wifi_mgmt` in the Zephyr documentation.
+
+For more details on the usage of the scan API, see :ref:`wifi_scan_sample` sample.
+
+Memory optimizations
+********************
+
+Scan-only builds are optimized for memory compared to regular Wi-Fi builds (for example, by supporting :abbr:`STA (Station)` mode).
+This is achieved by:
+
+* Compiling out the WPA supplicant module
+
+* Compiling out non-scan related functionality in the Wi-Fi driver
+
+* Removing the need for network data buffers
+
+You can use the :kconfig:option:`CONFIG_WIFI_MGMT_SCAN_SSID_FILT_MAX` Kconfig option to fine-tune the memory size.
+This option can set the maximum number of SSIDs that can be specified for SSID filtering.
+
+Power and Timing optimizations
+******************************
+
+The scan controls can be used to achieve power and scan time optimizations on the nRF70 Series ICs.
+The default scan operation is a wildcard scan (all SSIDs) across all supported bands and all regulatory allowed channels.
+
+For world regulatory domain (default), a typical scan (defined below) takes around 4.6 s and draws around 55 mA current.
+
+* 2.4 GHz - 940 ms
+
+  Channels 1 - 11: Active scan (Dwell time: 50 ms)
+
+  Channels 12-14: Passive scan (Dwell time: 130 ms)
+
+* 5 GHz - 3640 ms
+
+  Channels 36-48, 52-64, 100-144, 149-177: Passive scan (Dwell time: 130 ms)
+
+For other regulatory domains, for example US, the scan time varies as per its regulatory rules.
+
+The following controls can fine-tune the time taken and power consumed by a scan operation:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
+     - Configuration parameters
+   * - Band filtering
+     - | Build time:
+       | :kconfig:option:`CONFIG_WIFI_MGMT_SCAN_BANDS`
+       | Run time:
+       | :c:enumerator:`bands`
```

Comment:
Shouldn't these be clickable links (in this case pointing to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/connectivity/networking/api/wifi.html#c.wifi_scan_params.bands)